### ISSUE TITLE
Strafe left and right are swapped between JK and MOTS

### ISSUE THREAD

**Issue by StrikerMan780** [shinyquagsire23/OpenJKDF2#245]
Seems if I have both JK and MOTS running from the same folder/player profiles, strafe left and right are swapped. Meaning, I have to rebind the two keys, or they'll end up backwards. (A ends up strafe right, and D ends up strafe left.)

**Comment by shinyquagsire23** [comment]
Yeah it's a thing, idk why they swapped them but they did in MoTS
